buffer/variables.tf

```diff
@@ -125,3 +126,15 @@ variable "billing_account_ids" {
   description = "List of Google billing account ids to allow Resource Buffer Service to use"
   default     = []
 }
+
+variable "parent_folder_id" {
+  type        = string
+  description = "Parent folder under which to create all pool-specific folders."
+  default     = ""
```

Comment:
worth extended description that empty string has special meaning to not create a folder.

Reply:
Done.